kettle/Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-IMG = gcr.io/k8s-testimages/kettle
+IMG = gcr.io/k8s-gubernator/kettle
```

Comment:
I feel like It'd be preferable to keep it in k8s-testimages? It's where most (if not all?) of our other test-infra-built images land

Reply:
Ok I will need to play around with the push because rn `make push` puts them in guber